main.js

```diff
@@ -12,30 +14,30 @@ class store {
     let booksCollection;
     if (localStorage.getItem('booksCollection') === null) {
       booksCollection = [];
-  }
-    else {
+    } else {
       booksCollection = JSON.parse(localStorage.getItem('booksCollection'));
     }
     return booksCollection;
   }
+
   static setbook(book) {
     const booksCollection = store.getbook();
     booksCollection.push(book);
     localStorage.setItem('booksCollection', JSON.stringify(booksCollection));
   }
 
-  static deletebook(title){
+  static deletebook(title) {
     const booksCollection = store.getbook();
     booksCollection.splice(title, 1);
     localStorage.setItem('booksCollection', JSON.stringify(booksCollection));
   }
-};
+}
 
 class ui {
   static displayBooksCollection() {
     const booksCollection = store.getbook();
     const booklist = document.querySelector('.bookList');
-    booksCollection.forEach(book => {       
+    booksCollection.forEach((book) => {
       const bookFrame = document.createElement('div');
       bookFrame.classList.add('bookFrame');
       bookFrame.innerHTML = `
```

Comment:
- [ ] The book list and the add book form should be on the same page as on the wireframe. Please fix this issue and make sure that your design follows the wireframe design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/182113407-948af44c-261c-419b-aba2-4a60e20927a8.png)

### Wireframe design 

![image](https://user-images.githubusercontent.com/83803414/182113459-a741fc40-b1ce-4c20-83f2-bf7910804e59.png)

Reply:
@ChaabaneAnas, It's not about the borders. For now, the book list and the add from are in two different sections. The form should be placed below the boom list.

Reply:
@ChaabaneAnas, I understand that but as code reviewers, we have some guidelines to follow.

Reply:
@thecodechaser, that doesn't make sense, 
when you require me to actually remove functionality,
 instead of adding one ...

Reply:
@ChaabaneAnas, When we perform code review, We have some guidelines for code and design. We strictly need to follow them that is what we're getting paid for. I understand your case is different. But we need to stick to the guidelines if we do not want to lose our job. 😄

Reply:
@thecodechaser, Yeah i think ur right im removing the functionality